src/tools/radosacl.cc

```diff
@@ -151,7 +151,7 @@ int main(int argc, const char **argv)
 
   ACLID id;
 
-  snprintf(id.id, ID_SIZE + 1, "%.16x", 0x1234);
+  snprintf(id.id, ID_SIZE + 1, "%.8x", 0x1234);
```

Comment:
@tchaikov As per my analysis the size matches correctly now with .8x and the warning goes away. Could you please give me detailed explanation of your comment ? Are you requesting changes ?

Thanks.